code/modules/mob/living/carbon/xenomorph/evolution.dm

```diff
@@ -199,6 +203,10 @@
 			to_chat(src, "<span class='warning'>Something in this place is interfering with our link to Queen Mother. We are unable to evolve to a psychic caste here!</span>")
 			return
 
+		if(isspacearea(get_area(src))) //In the unlikely event we find ourselves outside a hull in a space area, prevent evolving into a caste that can prevent hivemind collapse
+			to_chat(src, "<span class='warning'>We are unable to evolve into a Shrike in space!</span>")
+			return
```

Comment:
why thesse changes

Reply:
So Wraiths couldn't stall a game indefinitely by going outside of the ship

Reply:
remove these
they shouldnt be in space anyway because it kills them instantly anway